### PR TITLE
Add full time and update general confirmations(#1231).

### PR DESCRIPTION
**Pull request by dsummersl**
This is for https://github.com/openopps/openopps-platform/issues/1231

### CODE REVIEW

**Comment by mtorres253** [comment]
This is great, @dsummersl. When you're ready for us to review, click the "I'm done" button on the auction page https://micropurchase.18f.gov/admin/auctions/34

**Comment by mtorres253** [comment]
Hey @EricSchles and @rogeruiz if this looks good, let's merge. Let me know

File: assets/js/backbone/apps/tasks/show/controllers/task_show_controller.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ var LoginConfig = require('../../../../config/login.json');
 
 var fs = require('fs');
 var VolunteerTextTemplate = fs.readFileSync(__dirname + '/../templates/volunteer_text_template.html').toString();
+var VolunteerFullTimeTextTemplate = fs.readFileSync(__dirname + '/../templates/volunteer_full_time_text_template.html').toString();
```
Comment (rogeruiz) [inline assets/js/backbone/apps/tasks/show/controllers/task_show_controller.js]: Awesome. This is a good start to having more modular modal messages. 🎉

File: assets/js/backbone/apps/tasks/show/controllers/task_show_controller.js [hunk 2]
```diff
@@ -290,20 +291,30 @@ var TaskShowController = BaseView.extend({
         return;
       }
 
+      var hasFullTimeDetail = _.chain(this.model.attributes.tags)
+        .map(_.property('name'))
+        .indexOf('Full Time Detail')
+        .value() >= 0;
```
Comment (rogeruiz) [inline assets/js/backbone/apps/tasks/show/controllers/task_show_controller.js]: This check is a bit ~~confusing~~ hard to read, but it's also in many places in the codebase. It's okay for now. ✌️

File: assets/locales/en-US/translation.json [hunk 3]
```diff
@@ -44,10 +44,25 @@
...
+  },
+
   "volunteerModal": {
-    "notifySupervisor": "I understand it is my responsibility to confirm supervisor approval prior to committing to an opportunity.",
-    "cantCancel": "Once you offer to participate in an opportunity, you will not be able to cancel your commitment.",
-    "title": "Do you want to participate?",
+    "supervisorApproval": "I have my supervisor's approval to apply to this opportunity.",
+    "federalEmployee": "I am a federal employee (not a contractor).",
+    "specificLocation": "If this opportunity is for a specific location (i.e. it is not a virtual opportunity), I am located within the same commuting area.",
+    "title": "Are you eligible to participate?",
```
Comment (rogeruiz) [inline assets/locales/en-US/translation.json]: ~~Did we also need to update the `volunteerModal` copy in the #1231 issue?~~ I see it. Awesome work @dsummersl 👍

**Pull request merged**